storybook/stories/Button.stories.js

```diff
@@ -36,13 +36,13 @@ export const metaMaskOutline = () => (
 )
 
 export const uPortSolid = () => (
-  <UPortButton>
+  <UPortButton.Solid>
```

Comment:
Would it be better to change the button subcomponent types to a `variant="solid"` prop convention? We have issues in the docs site with importing components that have these dot notation subtypes - mdx can't process them so we have to do hacks to get around it.

Reply:
@MikeLockz Maybe it would, but that's not relevant to this change. I just wrote a story so I could render what is already there. I didn't change it to be this way.